go/ct/rlz/conditions.go

```diff
@@ -724,7 +732,20 @@ func (c *isAddressCold) Restrict(generator *gen.StateGenerator) {
 }
 
 func (c *isAddressCold) GetTestValues() []TestValue {
-	return IsAddressWarm(c.key).GetTestValues()
+	property := Property(c.String())
+	domain := boolDomain{}
+	restrict := func(generator *gen.StateGenerator, isWarm bool) {
+		key := c.key.GetVariable()
+		c.key.BindTo(generator)
+		if isWarm {
```

Comment:
could this ever be `true`? can the one in `IsAddressWarm` ever be false ? same for `Is/IsNotReadOnly`?

Reply:
The restrict function is now reused for isAddressWarm and isAddressCold, therefore both cases are possible.